frontend/src/modals/new-representation/NewRepresentationModal.types.ts

```diff
@@ -32,7 +32,7 @@ export interface GQLViewer {
 }
 
 export interface GQLEditingContext {
-  representationDescriptions: GQLRepresentationDescriptionConnection;
+  representationCreationDescriptions: GQLRepresentationDescriptionConnection;
```

Comment:
Just an innocent question: Does the `GQLRepresentationDescriptionConnection` should not be renamed to `GQLRepresentationCreationDescriptionConnection`?

Reply:
In theory, yes. Not that it would change anything functionally, but if we follow our coding rules to the letter, yes it should.